dash_html.py

```diff
@@ -0,0 +1,225 @@
+'''
+This file is forked from apps/dash-clinical-analytics/app.py under the following license
+
+MIT License
+
+Copyright (c) 2019 Plotly
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+
+Modifications are licensed under
+
+Apache License, Version 2.0
+(see ./LICENSE for details)
+
+'''
+
+import dash
+from dash import dcc, html
+
+from app_configs import SCENARIOS, MODEL_OPTIONS, SOLVER_OPTIONS, HTML_CONFIGS
+
+
+
+def description_card():
+    """
+    :return: A Div containing dashboard title & descriptions.
+    """
+    return html.Div(
+        id="description-card",
+        children=[
+            html.H1(HTML_CONFIGS['main_header']),
+            html.P(children=HTML_CONFIGS['welcome_instructions']),
+        ],
+    )
+
+def generate_control_card() -> html.Div:
+    """
+    This function generates the control card for the dashboard, which
+    contains the dropdowns for selecting the scenario, model, and solver.
```

Comment:
```suggestion
    """Generates the control card for the dashboard.

    Contains the dropdowns for selecting the scenario, model, and solver.
```

Reply:
Quite a few other docstrings could also be cleaned up a bit in a similar way.

Reply:
Updated the ones in app.py too, let me know if these look good: https://github.com/dwave-examples/job-shop-scheduling-cqm/pull/16/commits/d0a9e52493c794ac8da5473b036072e37bc04d91

Reply:
Thanks, LGTM. 👍